hub/ui-new/src/components/rating/Rating.tsx

```diff
@@ -141,44 +142,62 @@ const Rating: React.FC = (props: any) => {
   if (props.isAuthenticated === true) {
     login = <form onClick={updateRating}>
       <ul className="rate-area" >
-        <input type="radio" id="5-star"
-          name="rating" value="5" checked={fivechecked} />
+        <input type="radio" id="5-star" name="rating"
```

Comment:
could we use the same formatting as the block below? 

```react
<input ... />
<label ... > <StarIcon /> </label>
```

Reply:
It was giving an `eslint` error of limited chars in a line